src/app/modules/item/components/item-children-edit/item-children-edit.component.html

```diff
@@ -3,16 +3,29 @@
 
     <ng-container *ngSwitchCase="'ready'">
       <div class="section-chapter-wrapper">
-        <p-table *ngIf="data.length" [value]="data" [reorderableColumns]="true" (onRowReorder)="orderChanged()">
+        <p-table *ngIf="data.length" [value]="data" [reorderableColumns]="true" [(selection)]="selectedRows" (onRowReorder)="orderChanged()">
           <ng-template pTemplate="body" let-rowData let-index="rowIndex">
             <tr [pSelectableRow]="rowData" [pReorderableRow]="index">
               <td class="reorder-handle" [pReorderableRowHandle]="index">
                 <i class="fa fa-sort" [pReorderableRowHandle]="index"></i>
               </td>
+              <td><p-tableCheckbox [value]="rowData"></p-tableCheckbox></td>
               <td class="type">{{ rowData.type }}</td>
               <td class="main">{{ rowData.title }}</td>
             </tr>
           </ng-template>
+          <ng-template pTemplate="summary">
+            <div class="summary-container">
+              <p-tableHeaderCheckbox></p-tableHeaderCheckbox>
+              <span class="select-all">Select all</span>
+              <div class="export-tools">
+                <span class="remove" (click)="deleteSelected()">
+                  <span>Remove</span>
```

Comment:
Call it "Remove selected" and grey it out (and so disable it) when nothing is selected (as in pending request panel)

Reply:
Done

Reply:
It is not greyed-out, it is not displayed at all. I would prefer greyed-out.